src/connectors/pagination/Paginator.ts

```diff
@@ -31,11 +41,12 @@ class Paginator {
     return range({ start: first, end: last });
   }
 
-  nbPagesDisplayed(padding, total) {
-    return Math.min(2 * padding + 1, total);
-  }
-
-  calculatePaddingLeft(current, padding, total, totalDisplayedPages) {
+  public calculatePaddingLeft(
```

Comment:
This is rather an internal method no?
It's only used within this class (in `pages`) so it can be private?

Reply:
```suggestion
  private calculatePaddingLeft(
```